opencypher/ast/cypher.py

```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass
-from typing import Iterable, Optional, Sequence, Union
+from typing import Iterable, Optional, Sequence, Union as UNION
```

Comment:
We now have two things named `Union`: the AST and Python's typing value.

I decided it was better to have both than to name the new one `Union_` or `UnionAST`